src/commands/hardis/lint/unusedMetadatas.ts

```diff
@@ -0,0 +1,178 @@
+/* jscpd:ignore-start */
```

Comment:
file and command name must be lowercase: please name the file into **unusedmetadatas** :)